src/const_encoded_len.rs

```diff
@@ -0,0 +1,164 @@
+// Copyright (C) 2023 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Contains the [`ConstEncodedLen`] trait and compliance tests.
```

Comment:
```suggestion
//! Contains the [`ConstEncodedLen`] trait. 
```

These are User facing docs and don't need to mention testing.

Reply:
Right, i was planning to add some automatic conformance tests, but it is just one line so probably not worth it.